wrangler/src/txFees/get.js

```diff
@@ -8,7 +9,10 @@ export default async ({ request, env }) => {
 
   const { publicKey: authedPublicKey } = authTxToken(STELLAR_NETWORK, feeToken)
 
-  const { metadata: feeMetadata } = await TX_FEES.getWithMetadata(authedPublicKey)
+  const txFeesId = TX_FEES.idFromName(authedPublicKey)
+  const txFeesStub = TX_FEES.get(txFeesId)
```

Comment:
I'm assuming these two lines refer to the `TX_FEES` durable object that gets bound during deployment?